database/data/schemaHelpers.js

```diff
@@ -0,0 +1,43 @@
+const db = require('../index');
+
+module.exports = {
+  songSaver: (song, cb) => {
+    // TBD: write code to generate a single song instance
+    const stmt = `insert into songs (length, timestamp, isliked, songfile, title, artist, album, thumbnail) 
+               values (?, ?, ?, ?, ?, ?, ?, ?)
+              `;
+    const songVals = [
+      song.length,
+      song.timestamp,
+      song.isliked,
+      song.songfile,
+      song.title,
+      song.artist,
+      song.album,
+      song.thumbnail,
+    ];
+    db.queryAsync(stmt, songVals)
+      .then((results) => cb(null, results))
+      .catch((err) => cb(err));
+  },
+  playlistSaver: (songid, playlist, cb) => {
+    // TBD: write code to generate a single upNext play instance
+    const stmt = `insert into ${playlist} (songid) 
+               values (?)
+              `;
+    const songVal = [songid];
+    db.queryAsync(stmt, songVal)
+      .then((results) => cb(null, results))
+      .catch((err) => cb(err));
+  },
+  songGetter: (songid, cb) => {
+    const stmt = `SELECT * FROM songs WHERE id = ?
```

Comment:
Why does this "stmt" string has uppercase keywords but the others do not? It seems inconsistent.